### DISCUSSION TITLE
Ability to segment different people in owntracks

### DISCUSSION THREAD

**Discussion by ghost** [owntracks/android#1870] [Ideas & New Features]
Have been using OwnTracks for a while now. Loving it, seems to work pretty well and offers great functionality.
I do have a feature request.

- Ability to segment people from being able to see each other location.

I am currently running this with a friend, some family have offered me Life360 and I have refused, since I don't trust Life360. I was thinking I could put them onto my own OwnTracks recorder, but I don't want family to be able to see location of my said friend.

Don't know if it is hard to implement, but this could be a pretty nice feature (Unless it's already possible and I'm just not sure how to do it).

**Comment by ckrey** [comment]
If you use OwnTracks with a Quicksetup backend or via a manually configured MQTT backend, you can setup MQTT acl as describe here: https://owntracks.org/booklet/features/qs-config/#friend-specific-mqtt-acls

**Comment by jpmens** [comment]
We should add that this works very well with our apps (android and iOS), but there’s no segmentation in Recorder or Frontend.
